src/Defaults/MergeTag/Post/PostType.php

```diff
@@ -46,9 +34,9 @@ public function __construct( $params = [] ) {
 				'name'        => __( 'Post Type', 'notification' ),
 				'description' => 'post',
 				'example'     => true,
-				'group'       => WpObjectHelper::get_post_type_name( $this->post_type ),
+				'group'       => WpObjectHelper::get_post_type_name( $this->get_trigger_prop() ),
 				'resolver'    => function ( $trigger ) {
-					return $trigger->post_type;
+					return $trigger->{ $this->get_trigger_prop() };
```

Comment:
This is wrong, this merge tag should return the post_type from the trigger, a literal `post_type` prop, not evaluated name of that type